docs/useful-robot-commands.md

```diff
@@ -0,0 +1,130 @@
+# Common Robot Commands
+
+# Table of Contents
+* [Off Robot Commands](#off-robot-commands)
+   * [Wifi Disclaimer](#wifi-disclaimer)
+   * [Flashing the nano](#flashing-the-nano)
+   * [Flashing the powerboard](#flashing-the-powerboard)
+   * [Setting up nano](#setting-up-nano)
+   * [Robot Diagnostics](#robot-diagnostics)
+* [On Robot Commands](#on-robot-commands)
+   * [Systemd Services](#systemd-services)
+   * [Debugging Uart](#debugging-uart)
+   * [Redis](#redis)
+
+# Off Robot Commands
+
+## Wifi Disclaimer
+
+To use most of these commands you will either need to be on the tbots wifi network( no internet access ) or on a wifi with internet access(shawopen, ubc wifi) and connected to the jetson nano through ethernet tethering. 
+
+The IP address of the robots on the tbots network is `192.168.0.20<robot_id>` so for robot id `1` the ip is `192.168.0.201`. If you are using ethernet tethering you will need to use a network utility (tshark, wireshark, arp) to determine the ip address.
+
+## Flashing the nano
+
+This will stop the current systemd services, replace and restart them. Binaries that are used for systemd services are located in a folder on home(the default directory) called thunderbots_binaries.
```

Comment:
```suggestion
This will stop the current systemd services, replace and restart them. Binaries that are used for systemd services are located in a folder in `home/robot/` (the default directory) called `thunderbots_binaries`.
```

Reply:
minor nit